tests/CustomConstraintsTests.cc

```diff
@@ -363,19 +363,19 @@ TEST(CustomConstraintCorrectnessTest) {
   Vector3d r020 = CalcBodyToBaseCoordinates(
                     dbj.model,dbj.q,dbj.idB2,
                     Vector3d(0.,0.,0.),true);
-  Vector3d r030 = CalcBodyToBaseCoordinates(
-                    dbj.model,dbj.q,dbj.idB2,
-                    Vector3d(dbj.l2,0.,0.),true);
+  // Vector3d r030 = CalcBodyToBaseCoordinates(
+  //                   dbj.model,dbj.q,dbj.idB2,
+  //                   Vector3d(dbj.l2,0.,0.),true);
 
   SpatialVector v010 = CalcPointVelocity6D(
                         dbj.model,dbj.q,dbj.qd,dbj.idB1,
                         Vector3d(0.,0.,0.),true);
   SpatialVector v020 = CalcPointVelocity6D(
                         dbj.model,dbj.q,dbj.qd,dbj.idB2,
                         Vector3d(0.,0.,0.),true);
-  SpatialVector v030 = CalcPointVelocity6D(
-                        dbj.model,dbj.q,dbj.qd,dbj.idB2,
-                        Vector3d(dbj.l2,0.,0.),true);
+  // SpatialVector v030 = CalcPointVelocity6D(
+  //                       dbj.model,dbj.q,dbj.qd,dbj.idB2,
+  //                       Vector3d(dbj.l2,0.,0.),true);
```

Comment:
If these are not used, why keep them commented out?

Reply:
Hi Martin,
as we are about to redo the unit test, I left the things commented for the next to pick it up.
Best
Manuel